src/main/java/com/baidu/hugegraph/structure/schema/IndexLabel.java

```diff
@@ -243,4 +247,21 @@ public long taskId() {
             return this.taskId;
         }
     }
+
+    @JsonIgnoreProperties({"properties", "user_data"})
+    public static class IndexLabelV49 extends IndexLabel {
+
+        public IndexLabelV49(IndexLabel indexLabel) {
+            super(indexLabel.name);
+            this.baseType = indexLabel.baseType;
+            this.baseValue = indexLabel.baseValue;
+            this.indexType = indexLabel.indexType;
+            this.fields = indexLabel.fields;
```

Comment:
check indexLabel.userdata() is empty

Reply:
The indexLabel.userdata() can be not empty